src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import './operatorsHbarChart.scss'
+
+const colorsByCompannies: { [index: string]: string } = {
+  'אגד תעבורה': 'rgb(47, 146, 80)',
```

Comment:
can use the operator enum instead of their display name?

Reply:
do you mean to change the keys to `agency_id`s?
I don't think it would be so readable

Reply:
see `src/model/operator.ts`, I don't want magic strings that are not validated by ts at the very least

Reply:
so lets put it here -
 open-bus-map-search/node_modules/open-bus-stride-client/agencies/agencyList.ts